docs/release_notes.rst

```diff
@@ -15,6 +15,12 @@ Data Cleaning
   :issue:`3340` and PR :pr:`3419`. This change also fixed a bug that was preventing
   other columns harvested with a special process from being saved.
 
+Schema Changes
+^^^^^^^^^^^^^^
+* Added ``balancing_authority_code_eia`` and ``sector_id_eia`` into the
+  :ref:`core_eia860m__changelog_generators` table. This required some cleanup of the BA
+  codes, which were not standardized. See issue :issue:`3437` and PR :pr:`3442`.
```

Comment:
do you mean that the BA codes in 860m were not standardized in this table? If so i'm not sure that addition is super helpful in release notes because its not like they existed in the table beforehand. totally a nit and not a hill i will die on.

Reply:
I meant that the BA codes in the original EIA860M were not standardized. They were significantly messier than the BA codes reported elsewhere.  Like there were as many new funky codes in just this one table as we had collected from all the other tables.

Reply:
I'll clarify the note.

Reply:
the clarification is nice! i still don't think this is a helpful release note (if we had been including this column but not encoding it and then transitioned to doing the cleanup then i think it would make more sense). but again not a hill i will die on.